<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projet Trésorerie</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/icon.ico</normaloff>../resources/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionCharger"/>
    <addaction name="actionSauvegarder"/>
    <addaction name="actionSauvegarderEnTantQue"/>
    <addaction name="separator"/>
    <addaction name="actionFermerSession"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Édition</string>
    </property>
    <addaction name="actionCloturerLivre"/>
    <addaction name="actionGenererBilan"/>
    <addaction name="actionGenererReleve"/>
    <addaction name="actionGenererResultat"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionChargementContexteLancement"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCharger">
   <property name="text">
    <string>Charger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionSauvegarderEnTantQue">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Sauvegarder en tant que</string>
   </property>
  </action>
  <action name="actionFermerSession">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Fermer session</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionGenererBilan">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Générer bilan</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+B</string>
   </property>
  </action>
  <action name="actionGenererReleve">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Générer relevé recettes/dépenses</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="actionGenererResultat">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Générer résultat</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionCloturerLivre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clôturer le livre</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionChargementContexteLancement">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chargement du contexte au lancement</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
